### ISSUE TITLE
[BUG]: /sys/fs/cgroup has no file in busybox image

### ISSUE THREAD

**Issue by Ace-Tang** [alibaba/pouch#1247]
### Ⅰ. Issue Description

Running a container with pouch, and check `ls /sys/fs/cgroup` is null
```
ace@www:~/go/src/github.com/docker/docker  (v1.12.6) time-> 六 4月 28 00:12:17
$ sudo pouch run -it registry.hub.docker.com/library/busybox:1.28 sh
/ # ls /sys/fs/cgroup/
/ # exit
ace@www:~/go/src/github.com/docker/docker  (v1.12.6) time-> 六 4月 28 00:12:43
$ sudo pouch run -it reg.docker.alibaba-inc.com/base/busybox:latest sh
/ # ls /sys/fs/cgroup/
/ # 
```

while I run a container with moby, I found that `ls /sys/fs/cgroup` has files
```
ace@www:~/go/src/github.com/docker/docker  (v1.12.6) time-> 五 4月 27 23:21:45
$ docker run -d  reg.docker.alibaba-inc.com/base/busybox:latest top
06855cad26a3116cd788a60c952e3d010a5e2b2e19d93d382818210215a1b33a

ace@www:~/go/src/github.com/docker/docker  (v1.12.6) time-> 五 4月 27 23:22:01
$ docker exec -it 06855cad26a3 sh
/ # ls /sys/fs/cgroup/
blkio             cpu,cpuacct       cpuset            freezer           memory            net_cls,net_prio  perf_event        systemd
cpu               cpuacct           devices           hugetlb           net_cls           net_prio          pids
```

I wonder that is images pulled by pouch and moby are different .

### Ⅳ. How to reproduce it (as minimally and precisely as possible)

1.
2.
3.

### Ⅵ. Environment:

- pouch version (use `pouch version`):
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

**Comment by fuweid** [comment]
Hi @Ace-Tang , is the #1227 PR used to fix this issue? If not, I would like to take a look.

**Comment by Ace-Tang** [comment]
@fuweid , no, the pr #1227 is influenced by this issue, I will be glad if you can help this.

**Comment by fuweid** [comment]
Got it. Thanks

**Comment by Ace-Tang** [comment]
hi, @fuweid , do you have some update?

**Comment by fuweid** [comment]
@Ace-Tang still investigating...

**Comment by fuweid** [comment]
Fixed by #1314
